### DISCUSSION TITLE
Welcome to RasgoQL Discussions!

### DISCUSSION THREAD

**Discussion by Nick-Harvey** [rasgointelligence/RasgoQL#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by agatasiwekjendobrezakopane4eve** [comment]
Brilliant !! Just what I was thinking about this morning what I need / want!! Will be using it no doubt soon !! Hopefully.

**Comment by venkat0608** [comment]
If my source storage is Elasticsearch, how can I use rasgo to build features?

**Comment by mashhype** [comment]
I get this error when I try to pip install rasgoql[snowflake]:  no matches found: rasgoql[snowflake]

Prior to running this, I successfully downloaded the snowflake connector...

**Comment by Nick-Harvey** [comment]
I ran into the same thing. The fix is to use quotes 'rasgoql[snowflake]'
